src/rest_api/amoc_api_helpers_execution.erl

```diff
@@ -0,0 +1,83 @@
+%%==============================================================================
+%% Copyright 2020 Erlang Solutions Ltd.
+%% Licensed under the Apache License, Version 2.0 (see LICENSE file)
+%%==============================================================================
+-module(amoc_api_helpers_execution).
+
+%% API
+-export([start/1, stop/0, add_users/1, remove_users/1, update_settings/1]).
+
+-type body() :: #{binary() => any()}.
+-type ret_value() :: {ok, any} | {error, any()}.
+
+-spec start(body()) -> ret_value().
+start(#{<<"scenario">> := ScenarioName} = Body) ->
+    case
+        amoc_api_helpers_scenario_info:is_loaded(ScenarioName) of
+        {true, Scenario} ->
+            Users = maps:get(<<"users">>, Body, 0),
+            SettingsMap = maps:get(<<"settings">>, Body, #{}),
+            case read_settings(SettingsMap) of
+                {ok, Settings} ->
+                    amoc_dist:do(Scenario, Users, Settings);
+                {error, _} = Err -> Err
+            end;
+        false ->
+            {error, no_such_scenario}
+    end;
+start(_) ->
+    {error, invalid_body}.
```

Comment:
Does it make sense to extend the error description?
```suggestion
start(_) ->
    {error, {invalid_body, "missing `scenario` filed in the request's body"}}.
```

Reply:
this is stub, with the current version of amoc_rest we will never hit this code.  amoc_rest framework verifies input and rejects it if it doesn't correspond to a schema. 

the only reason to keep this code is to give a quick reference where the problem occurred, if there's a mismatch between interface declaration (amoc_rest) and implementation (amoc).